data/mods/Magiclysm/effects/effects.json

```diff
@@ -1271,11 +1271,12 @@
     "type": "effect_type",
     "id": "effect_flame_immunity",
     "name": [ "Flame Immunity" ],
-    "desc": [ "You are immune to heat and fire." ],
+    "desc": [ "You are immune to heat and fire" ],
     "rating": "good",
     "show_in_info": true,
-    "removes_effects": [ "onfire", "blisters" ],
-    "enchantments": [ "ench_heat_and_fire_immunity" ]
+    "blocks_effects": [ "onfire", "blisters" ],
```

Comment:
Changing this from removes to blocks means that if you apply this effect to someone who is on fire, they aren't extinguished. Is that intended?

Reply:
No, I will fixed that.

Reply:
But I keep blisters as blocks. Cause if already got blister it is not gonna just disappeared.

Reply:
Should that be true for a Vulcanist? This is magic, after all--it could just disappear.

Reply:
It is very unlikely having blister when receiving attunement, I think it is safe to ignore this edging case.